### DISCUSSION TITLE
Integrate SimpleFIN with Firefly using N8N

### DISCUSSION THREAD

**Discussion by darsh12** [firefly-iii/firefly-iii#8874] [General]
I recently picked up firefly again to start managing my accounts and I knew that firefly has Plaid connector, but I wanted something easy to maintain and customize. So I came about SimpleFIN and found there is an open issue #5396 to add this to the data importer. However, since I wanted it now, I setup the integration of firefly and SimpleFIN using N8N. It can also categorize two transactions as a transfer based upon a criteria that is modifiable. 

The gist can be found here https://gist.github.com/darsh12/4e93abfc6a6307c6ae0933e325f2084f.

I have also written a blog describing the workflow https://unhackable.lol/firefly/

**Comment by JC5** [comment]
Cool! Very nice work!

**Comment by adamcstephens** [comment]
Looks like someone made a Python script here https://github.com/Sc00tz/SimpleFIN-to-Firefly_III
